scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc

```diff
@@ -40,6 +40,21 @@ include::modules/ztp-configuring-ptp-fast-events.adoc[leveloffset=+1]
 
 * For more information about how to install the AMQ Interconnect Operator, see xref:../../monitoring/using-rfhe.adoc#hw-installing-amq-interconnect-messaging-bus_using-rfhe[Installing the AMQ messaging bus].
 
+include::modules/ztp-add-local-reg-for-sno-duprofile.adoc[leveloffset=+1]
+
+include::modules/ztp-configuring-disk-partitioning.adoc[leveloffset=+2]
```

Comment:
Since this is a topic that is useful for ZTP use cases beyond using a local registry with single-node OpenShift, you could add it to the ZTP advanced install assembly as outlined here: https://github.com/openshift/openshift-docs/pull/53083/files#diff-48ea496dc82a53051e13de9bf557531877d677c8105dcdcb6a357fab8a985ad2

This [version](https://github.com/openshift/openshift-docs/pull/53083/files#diff-c0c2580378cc5e557211fa281f26dbe3efdd1da52e33bc7ef3873f9f10639307) of the module is generalized for this purpose.

Reply:
Outside the scope of this PR. We can create a PR in the future to address generalizing these topics.